tests/unit/test_utils.py

```diff
@@ -54,3 +54,24 @@ def test_safe_stringify(value):
     safe_value = utils.safe_stringify(value)
     assert safe_value == "1"
     assert type(safe_value) == str
+
+
+def test_payload_wrapper_methods():
```

Comment:
I'd honestly be fine not having this test since we don't actually define any methods in PayloadWrapper now, but it doesn't hurt to have I guess

Reply:
Yeah, I feel the same. It's kind of a 🤷 for me, but it does validate that the rewrite to `UserDict` didn't break anything? Not that I was exactly worried... I left it there, but if it were removed in the future that would be fine too.